environments/hca-prod/terraform/buckets.tf

```diff
@@ -247,14 +247,6 @@ resource google_service_account_iam_binding hca_workload_identity_binding {
   members            = ["serviceAccount:${local.prod_project_id}.svc.id.goog[hca/argo-runner]"]
 }
 
-resource google_service_account_iam_binding hca_dagster_workload_identity_binding {
```

Comment:
(this resource is already set up in `dagster.tf` - i'm monitoring plans to make sure that this doesn't do anything unexpected.)